curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8ab0e27cbaf0b54ba8a42.md

```diff
@@ -0,0 +1,171 @@
+---
+id: 62a8ab0e27cbaf0b54ba8a42
+title: Step 51
+challengeType: 0
+dashedName: step-51
+---
+
+# --description--
+
+Just like array values, object properties are separated by a comma. Add a comma after your `name` property, and add a `button text` property with the value of an empty array. Note that because the property name has more than one word, you'll need to surround it in quotes.
```

Comment:
```suggestion
Just like array values, object properties are separated by a comma. Add a comma after your `name` property and add a `button text` property with the value of an empty array. Note that because the property name has more than one word, you'll need to surround it in quotes.
```